sdks/python/apache_beam/transforms/sideinputs_test.py

```diff
@@ -309,6 +316,65 @@ def test_flattened_side_input(self):
     assert_that(results, equal_to(['a', 'b']))
     pipeline.run()
 
+  @attr('ValidatesRunner')
+  def test_multi_triggered_gbk_side_input(self):
+    """Test a GBK sideinput, with multiple triggering."""
+    options = PipelineOptions()
+    options.view_as(StandardOptions).streaming = True
+    p = TestPipeline(options=options)
+
+    main_stream = (p
+                   | 'main TestStream' >> TestStream()
+                   .advance_watermark_to(3)
+                   .add_elements(['a1'])
+                   .advance_watermark_to(8)
+                   .add_elements(['a2'])
+                   | 'main windowInto' >> beam.WindowInto(
+                       window.FixedWindows(5),
+                       trigger=trigger.AfterWatermark(
```

Comment:
Why is there a trigger on the main stream?

Reply:
Without trigger on main stream, how do we expect multiple outputs in the final result? 

The current sequence is: 
1) side input triggers
2) main triggers  ==> leads to output #1
3) side input triggers (thus different side input)
4) main triggers ==> leads to output #2

Does that make sense?  or did I miss anything?